### ISSUE TITLE
Complete setup for FED-BOM squad on-call

### ISSUE THREAD

**Issue by robrap** [edx/edx-arch-experiments#118]
This ticket is for getting Fed-BOM set up for on-call. Currently, some features that Fed-BOM owns (that used to be owned by CE), are alerting to Arch-BOM. This includes some Account MFE alerts, that are potentially related to an active issue or where the alerts need to be tuned. Arch-BOM on-call can help with initial questions. Ongoing questions can be answered in `#tech-ocm-observability` or `#tech-dev-edx` (or SRE or embedded-SRE channels).

Here are a potential steps:
- [x] Ensure entire team can access New Relic through OneLogin.
- [x] Ensure entire team can access OpsGenie through OneLogin.
- [x] Work with Arch-BOM on-call (sync and/or async) to get OpsGenie set up. (We may want to add this ticket to both boards at that time.)
- [ ] TBD (To be determined)

**Comment by robrap** [comment]
@timmc-edx: If there is a ticket for Account MFE alert issues, can you link it here?
@jmbowman: FYI: I moved this to the top, but please prioritize as you see fit. Thank you.

**Comment by timmc-edx** [comment]
There's no ticket for just alert issues, but there is one for the errors: https://github.com/openedx/frontend-app-account/issues/701

(Alert policy https://onenr.io/02R5NWbvkjb is currently set to 30 minutes for 20% error rate, and should probably be restored to 5 minutes.)

**Comment by robrap** [comment]
Thanks @timmc-edx. I copied your note about the alert policy change to the other ticket, assuming that it is the blocker on restoring.

**Comment by robrap** [comment]
@abdullahwaheed: What is the status of this? It hasn't moved in a long time. Thank you.

**Comment by abdullahwaheed** [comment]
@robrap we don't have OpsGenie as of now. one of my GoCD permission ticket is in process as well

**Comment by robrap** [comment]
Hello @abdullahwaheed. It seems like this ticket is now unblocked. Please reach out with questions, and/or we can set up a meeting to review setup. Take care.

**Comment by abdullahwaheed** [comment]
Hi @robrap. Yes now we have OpsGenie access. It would be great if we can schedule a meeting to discuss this

**Comment by robrap** [comment]
> Hi @robrap. Yes now we have OpsGenie access. It would be great if we can schedule a meeting to discuss further

@rgraber will be working with your team to support setup. You all can work out when and if to meet in Slack.

Also, the following doc contains some useful information for getting individuals and teams set up in OpsGenie, including configuring the mobile app and testing individual (or team) alert notifications: https://2u-internal.atlassian.net/wiki/spaces/SRE/pages/19239418/How+to+migrate+OpsGenie+from+edX+to+2U+instance.

Fed-BOM (and @abdullahwaheed): Please have all members ensure that you can actually get alerted properly. It would be good to have everyone confirm somewhere. 

@rgraber: This is the doc I mentioned. Can you find a home for the parts that are still relevant and then archive this document (unless you simply rework and re-title this document)? Thank you!

**Comment by robrap** [comment]
@rgraber: Also, I added you an an assignee for this ticket and added it as in-progress on the Arch-BOM board.

**Comment by rgraber** [comment]
Closing this ticket as I think we have successfully set FED-BOM up. @abdullahwaheed If there are any more issues with on-call or you want more help setting up feel free to open another issue.
